### PR TITLE
How to take over Abandoned feedstock

### LINKED ISSUE #1431: [DOC] Explain how to take over an abandoned feedstock
I'm actually not sure if this is already documented but I at least cannot find it anywhere.

It would be good to have the process of taking over an abandoned feedstock documented somewhere.

The workflow for this is generally:

1. Create a PR and add your GitHub username to the `meta.yaml` (The current maintainer should not be removed, only if they later request so)
2. Rerender with smithy
3. Ask core by pinging them or in gitter to merge the PR
4. Fix any pending PRs and update the feedstock, if necessary.

Prerequisites:

A feedstock is generally considered abandoned if the maintainer is not on GitHub anymore or if multiple bot PRs are open. An exact definition can be found here: https://github.com/conda-forge/cfep/pull/15 although that is not yet accepted. Therefore that part is currently on a case-by-case basis and decided by core.

### PR DESCRIPTION
**Pull request by PrernaSingh587**
Fixes #1431 

@BastianZim , Please let me know where it needs improvement. 
Thanks.

### CODE REVIEW

**Comment by BastianZim** [comment]
Looks great, thank you!

**Comment by BastianZim** [comment]
Gently pinging this @conda-forge/core

**Review by dopplershift — APPROVED** [review]

**Comment by BastianZim** [comment]
@dopplershift Would you mind also merging it?

File: src/maintainer/updating_pkgs.rst [hunk 1]
```diff
@@ -218,6 +218,18 @@ package (an example would be to re-render the feedstock to support new Python ve
...
 If you believe a feedstock should be archived, please contact `@conda-forge/core <https://github.com/orgs/conda-forge/teams/core>`__.
 
+Taking over Abandoned Feedstock
+================================
+
+A feedstock is generally considered abandoned if it has no or unresponsive maintainers. Also, it may have multiple bot PRs opened. These feedstocks can be archived or need new maintainers.
+
+To take over the abandoned feedstock and become their new maintainers :
+
+1. Create a `Pull Request <https://conda-forge.org/docs/misc/00_intro.html>`__ to the feedstock repository and add your username to the ``meta.yaml``. (Make sure not to remove the current maintainer’s name from it unless requested by them)
```
Comment (isuruf) [inline src/maintainer/updating_pkgs.rst:228]: Need to add `[ci skip] [skip ci] ***NO_CI***` to commit message and pull request title.

Comment (PrernaSingh587) [reply src/maintainer/updating_pkgs.rst]: Added. Anything else on this? Please let me know. @isuruf

**Comment by BastianZim** [comment]
Hi @viniciusdc Just wanted to quickly check here if it's possible to get this merged still?

**Pull request closed without merging**